src/components/Datatable/hooks/useDataFetch.ts

```diff
@@ -1,21 +1,21 @@
 import { useEffect } from 'react';
+import { pick, propEq, when } from 'ramda';
 
 import { DatatableStore } from '../Datatable.store';
 import { OnDataFetchFn } from '../Datatable.types';
 
 export const useDataFetch = <D>(onDataFetch: OnDataFetchFn<D>): void => {
   useEffect(() => {
     const unsubscribe = DatatableStore.subscribe(
-      (s) => ({
-        pageIndex: s.pageIndex,
-        pageSize: s.pageSize,
-        sortBy: s.sortBy,
-        filters: s.filters,
-        query: s.query,
-      }),
-      (params) => {
-        onDataFetch(params);
-      },
+      pick([
```

Comment:
I already mentioned it to someone, I'm definitely in favour to use ramda here, but we should do it consistently for all occurrences. We should be consistent throughout codebase and not to have half done by ramda and the other half differently.

Reply:
I agree 100%, I'm for Ramda but consistency > personal preferences. I only went with the change here because the entire file is now Ramda based, way I see it, is that if we can get a file completely migrated it's already a win as it's very unlikely we'll ever have the chance to do it all at once. what do you think?

Reply:
Can you promise me that you'll migrate all to Ramda? :D I get it could you please create a ticket in the DS epic so we can tackle this and don't forget about it?

Reply:
haha I'll put it up as one of the goals, and absolutely https://zitenote.atlassian.net/browse/FEP-1818